types/aframe/index.d.ts

```diff
@@ -1,21 +1,22 @@
-// Type definitions for AFRAME 1.0
+// Type definitions for AFRAME 1.2
 // Project: https://aframe.io/
 // Definitions by: Paul Shannon <https://github.com/devpaul>
 //                 Roberto Ritger <https://github.com/bertoritger>
 //                 Trygve Wastvedt <https://github.com/twastvedt>
+//                 Marc Buils <https://github.com/metapins>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.6
 
 /**
  * Extended tests and examples available at https://github.com/devpaul/aframe-experiments.git
  */
 
+import * as anime from 'animejs';
```

Comment:
I don't think the package is correct. Please see this file
https://github.com/aframevr/aframe/blob/master/src/index.js#L102

Reply:
It seems to be a fork to animejs so I supposed interface is same (I didn't find the git repository). What do you think ?
https://www.npmjs.com/package/super-animejs